crates/ty_project/src/metadata.rs

```diff
@@ -281,7 +309,7 @@ impl ProjectMetadata {
 }
 
 #[derive(Debug, Error)]
-pub enum ProjectDiscoveryError {
+pub enum ProjectMetadataError {
```

Comment:
renamed to be more generic - includes errors encountered during discovery + errors when discovery is skipped and it's loaded from a specific config file